dds/DCPS/transport/tcp/TcpInst.cpp

```diff
@@ -68,6 +68,9 @@ OpenDDS::DCPS::TcpInst::load(ACE_Configuration_Heap& cf,
   GET_CONFIG_VALUE(cf, trans_sect, ACE_TEXT("active_conn_timeout_period"),
                    this->active_conn_timeout_period_, int)
 
+  GET_CONFIG_VALUE(cf, trans_sect, ACE_TEXT("requires_cdr_encapsulation"),
```

Comment:
As a function name I think this was fine at the time, but I'm not sure about using it as the name of the config option. This is because none of the transport except RTPS really require encapsulation from our perspective. But I guess the user is requiring to get XCDR, then maybe it is a good name? You don't have to do anything about this, I'm mainly just thinking out loud.